mount/mount_linux.go

```diff
@@ -147,6 +149,10 @@ func (mounter *Mounter) doMount(mounterPath string, mountCmd string, source stri
 	command := exec.Command(mountCmd, mountArgs...)
 	output, err := command.CombinedOutput()
 	if err != nil {
+		if err.Error() == errNoChildProcesses {
```

Comment:
this package is unused now, see https://github.com/kubernetes/utils/blob/master/mount/README.md for the currently used location of this code

Reply:
Aah, I'll close this PR and open a new one there. Thanks!